Lib/fontTools/pens/freetypePen.py

```diff
@@ -18,13 +18,13 @@
 from freetype.ft_enums import FT_OUTLINE_NONE, FT_OUTLINE_EVEN_ODD_FILL, FT_PIXEL_MODE_GRAY
 from freetype.ft_errors import FT_Exception
 
-from fontTools.pens.basePen import BasePen
+from fontTools.pens.basePen import BasePen, PenError
 from fontTools.misc.roundTools import otRound
 from fontTools.misc.transform import Transform
 
 Contour   = collections.namedtuple('Contour', ('points', 'tags'))
 LINE      = 0b00000001
-CURVE     = 0b00000011
```

Comment:
ah so all `LINE = CURVE = QCURVE = 0b00000001` basically, might be worth making it explicit in the code